docs/howtos/read_eq_prof.rst

```diff
@@ -0,0 +1,59 @@
+====================================================
+Generate an input file from equilibrium and profiles
+====================================================
+
+Here a step-by-step guide on how to generate an input file
+for any gyrokinetic codes supported in ``pyrokinetics``,
+starting from equilibrium and profile files.
+
+
+Let's first import ``pyrokinetics`` and define our equilibrium and input files. 
+
+.. code-block:: python
+   >>> from pyrokinetics import Pyro, template_dir
+   >>> eq_file = template / "test.geqdsk"
+   >>> kinetics_file = template_dir / "jetto.cdf"
+
+The equilibrium file ``test.geqdsk`` and the kinetics file ``jetto.cdf``
+are stored in the template foder and used here as an example.
+We now load these files into ``pyrokinetics``:
+
+.. code-block:: python
+
+   >>> pyro = Pyro(
+        eq_file=eq_file,
+        kinetics_file=kinetics_file,
+	kinetics_type="JETTO",
+        kinetics_kwargs={"time": 550},
+    )
+
+
+During initialization, ``Pyro`` calls ``read_equilibrium`` from
```

Comment:
I think we need a single back tick to link to the object

Reply:
Done.